tool/grind.dart

```diff
@@ -44,7 +44,6 @@ Future<void> generateLocalizations() async {
     '--preferred-supported-locales=["en"]',
     '--use-deferred-loading',
```

Comment:
Are these arguments still needed now with the l10n.yaml file? Or is it simply enough to just run gen_l10n.dart?

Reply:
The command itself shouldn't be necessary at all anymore, since localizations generation is baked into Flutter run, restart, reload, pub get, analyze and test. 

The only reason I've left it in for now is because there's an issue with the gen_l10n tool's having the incorrect dependencies when switching between flutter versions that I still need to fix. Running the grinder command will always work, whereas the synthetic package generation may fail unless `flutter update-packages` is run.